lib/libvfio-user.c

```diff
@@ -508,16 +508,16 @@ handle_dma_map(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg,
         dma_map->flags &= ~VFIO_USER_F_DMA_REGION_WRITE;
     }
 
-    if (dma_map->flags == VFIO_USER_F_DMA_REGION_MAPPABLE) {
+    if (dma_map->flags != 0) {
```

Comment:
I think we must be strict and fail if client passes unknown flags?

Reply:
Ah I forgot the `return` statement.